docs/develop/03_workshop/04_attester/01_account.md

```diff
@@ -39,6 +39,15 @@ A person can memorize it, and use it later to re-generate their keypairs and add
 To generate an account, we use the `addFromMnemonic()` function on the [`KiltKeyringPair`](https://kiltprotocol.github.io/sdk-js/interfaces/_kiltprotocol_types.KiltKeyringPair.html) generated via the SDK.
 The mnemonic is generated with the polkadot function `mnemonicGenerate()`, which generates a 12-word mnemonic.
 
+:::info polkadot.js
+
+The KILT SDK is built on top of the [polkadot.js](https://polkadot.js.org/) library therefore you will find various uses of it in this workshop.
+It provides a set of tools to interact with the KILT blockchain and other substrate based blockchains, provides cryptographic primitives and a serialization framework.
```

Comment:
It was a bit hard to follow. 

My proposal:

```suggestion
It provides a set of tools to interact with the KILT blockchain and other substrate based blockchains
In addition the polkadot-js library offers cryptographic primitives and a serialization framework to decode data received from the blockchain.
```

ChatGPTs proposal:
```suggestion
This JavaScript library includes a collection of tools for interacting with the KILT blockchain and other Substrate-based blockchains. Moreover, it offers cryptographic primitives and a serialization framework to facilitate blockchain development.
```

Reply:
One-sentence-one-line tho:

```suggestion
This JavaScript library includes a collection of tools for interacting with the KILT blockchain and other Substrate-based blockchains.
Moreover, it offers cryptographic primitives and a serialization framework to facilitate blockchain development.
```